src/ECER.Tests/Integration/Utilities/DataverseSdk/QueryTests.cs

```diff
@@ -33,7 +33,6 @@ public void WhereNotIn_Filtered()
     {
         var statuses = new[] { ecer_Application_StatusCode.Draft, ecer_Application_StatusCode.Complete };
         var query = dataverseContext.ecer_ApplicationSet.WhereNotIn(a => a.StatusCode!.Value, statuses).ToList();
-        query.ShouldNotBeEmpty();
```

Comment:
I fixed the test in another PR so you should merge master into your branch. In general, this is not a good practice to arbitrarily delete assertions that were there.